include/superior_mysqlpp/types/datetime.hpp

```diff
@@ -44,24 +69,41 @@ namespace SuperiorMySqlpp
         Datetime(Datetime&&) = default;
         Datetime& operator=(const Datetime&) = default;
         Datetime& operator=(Datetime&&) = default;
+        
+        /**
+         * @brief Default destructor.
+         */
         ~Datetime() = default;
 
+        /**
+         * @brief Returns the fraction of second stored in Datetime.
+         */
         auto getSecondFraction() const
         {
             return buffer.second_part;
         }
 
+        /**
+         * @brief Sets the fraction of second stored in Datetime.
+         */
         template<typename U>
         void setSecondFraction(U&& secondFraction)
         {
             buffer.second_part = std::forward<U>(secondFraction);
         }
 
+        /**
+         * @brief Returns the year, month and day stored in Datetime.
```

Comment:
When you write only simple description, `@brief` keyword is redundant. Don't use it in these cases please.
I prefer placing empty line between method brief and method parameters/return/throws for better readability.

Reply:
ACK